### PR TITLE
adding questions

### PR DESCRIPTION
**Pull request by mikiodehartj1**
### Alerts Rule Packs

- [ ] IF THIS CONTENT NEEDS TO BE RELEASED - is the package version in the package.json bumped?
- [ ] Does a related alert already exist, and should it be tweaked or added to instead?
- [ ] Test each query to make sure it works
- [ ] Look for hardcoded variables/parameter values in the query
- [ ] Consider Severity for Alerts
- [ ] Spellcheck
- [ ] Use all caps for J1QL keywords and relationship classes
- [ ] Upload the alerts rule pack JSON into JupiterOne to validate

### CODE REVIEW

File: rule-packs/mitre-attck-execution-attack-paths.json [hunk 1]
```diff
@@ -94,5 +94,257 @@
...
+      ],
+      "alertLevel": "HIGH"
+    },
+    {
+      "name": "execution-command-and-scripting-interpreter-execution-prevention-application-control-assessment",
+      "description": "M1038 - Use application control where appropriate. For example, PowerShell Constrained Language mode can be used to restrict access to sensitive or otherwise dangerous language elements such as those used to execute arbitrary Windows APIs or files (e.g., Add-Type).",
+      "queries": [
+        {
+          "name": "query0",
+          "query": "FIND azure_subscription THAT RELATES TO (azure_role_definition|azure_role_assignment) WITH actions ~= 'owner' OR actions ~= 'contributor'",
```
Comment (SeaBlooms) [inline rule-packs/mitre-attck-execution-attack-paths.json:128]: query duplicate from 'execution-cloud-administration-command-privileged-account-management-azure'

File: rule-packs/mitre-attck-execution-attack-paths.json [hunk 2]
```diff
@@ -94,5 +94,257 @@
...
+        {
+          "name": "query0",
+          "query": "FIND HostAgent WITH function = 'anti-malware' THAT !(PROTECTS | MANAGES | ASSIGNED | MONITORS) (Device | Host)",
+          "version": "v1"
+        }
+      ],
+      "alertLevel": "HIGH"
+    },
+    {
+      "name": "execution-command-and-scripting-interpreter-execution-prevention-application-control-assessment",
```
Comment (SeaBlooms) [inline rule-packs/mitre-attck-execution-attack-paths.json:123]: add '-azure' to name

File: rule-packs/mitre-attck-execution-attack-paths.json [hunk 3]
```diff
@@ -94,5 +94,257 @@
...
+        {
+          "name": "query0",
+          "query": "FIND azure_subscription THAT RELATES TO (azure_role_definition|azure_role_assignment) WITH actions ~= 'owner' OR actions ~= 'contributor'",
+          "version": "v1"
+        }
+      ],
+      "alertLevel": "HIGH"
+    },
+    {
+      "name": "execution-command-and-scripting-interpreter-execution-prevention-application-control-policy",
```
Comment (SeaBlooms) [inline rule-packs/mitre-attck-execution-attack-paths.json:135]: add '-azure' to name

**Review by SeaBlooms — CHANGES_REQUESTED** [review]
couple updates then we can get this pushed out

**Review by SeaBlooms — CHANGES_REQUESTED** [review]

File: rule-packs/mitre-attck-execution-attack-paths.json [hunk 4]
```diff
@@ -94,5 +94,257 @@
...
+      ],
+      "alertLevel": "HIGH"
+    },
+    {
+      "name": "execution-serverless-execution-user-account-management-aws-serverless-restrict",
+      "description": "M1018 - Remove permissions to create, modify, or run lambda resources from users that do not explicitly require them.",
+      "queries": [
+        {
+          "name": "query0",
+          "query": "FIND aws_iam_user (THAT RELATES TO aws_iam_role)? THAT ASSIGNED aws_iam_policy THAT RELATES TO as rel (aws_lambda_function|aws_lambda) WHERE rel.actions ~= ('lambda:CreateFunction' or 'lambda:DeleteFunction' or 'lambda:UpdateFunctionCode' or 'lambda:UpdateFunctionConfiguration')",
```
Comment (erincrawford) [inline rule-packs/mitre-attck-execution-attack-paths.json:212]: Would "lambda:*" need to be included?

File: rule-packs/mitre-attck-execution-attack-paths.json [hunk 5]
```diff
@@ -94,5 +94,257 @@
...
+      ],
+      "alertLevel": "HIGH"
+    },
+    {
+      "name": "execution-user-execution-malicious-file-user-training",
+      "description": "M1017 - Use user training as a way to bring awareness to common phishing and spearphishing techniques and how to raise suspicion for potentially malicious events.",
+      "queries": [
+        {
+          "name": "query0",
+          "query": "FIND (Person|User) THAT !COMPELTED Training WITH required = true",
```
[the remark this answers is not in the document]
Comment (mikiodehartj1) [reply rule-packs/mitre-attck-execution-attack-paths.json:320]: Corrected

File: rule-packs/mitre-attck-execution-attack-paths.json [hunk 6]
```diff
@@ -94,5 +94,257 @@
...
+      ],
+      "alertLevel": "HIGH"
+    },
+    {
+      "name": "execution-user-execution-malicious-link-user-training",
+      "description": "M1017 - Use user training as a way to bring awareness to common phishing and spearphishing techniques and how to raise suspicion for potentially malicious events.",
+      "queries": [
+        {
+          "name": "query0",
+          "query": "FIND (Person|User) THAT !COMPELTED Training WITH required = true",
```
[the remark this answers is not in the document]
Comment (mikiodehartj1) [reply rule-packs/mitre-attck-execution-attack-paths.json:296]: Corrected

[the remark this answers is not in the document]
Comment (mikiodehartj1) [reply rule-packs/mitre-attck-execution-attack-paths.json:296]: Corrected

File: rule-packs/mitre-attck-execution-attack-paths.json [hunk 7]
```diff
@@ -94,5 +94,257 @@
...
+      "name": "execution-software-deployment-tools-update-software",
+      "description": "M1051 - Patch deployment systems regularly to prevent potential remote access through Exploitation for Privilege Escalation.",
+      "queries": [
+        {
+          "name": "query0",
+          "query": "FIND aws_patch_group THAT HAS aws_instance",
+          "version": "v1"
+        }
+      ],
+      "alertLevel": "HIGH"
```
[the remark this answers is not in the document]
Comment (mikiodehartj1) [reply rule-packs/mitre-attck-execution-attack-paths.json:288]: Corrected

File: rule-packs/mitre-attck-execution-attack-paths.json [hunk 8]
```diff
@@ -94,5 +94,257 @@
...
+      "name": "execution-user-execution-malicious-image-audit-image",
+      "description": "M1047 - Audit images deployed within the environment to ensure they do not contain any malicious components.",
+      "queries": [
+        {
+          "name": "query0",
+          "query": "FIND Container THAT !HAS Assessment",
+          "version": "v1"
+        }
+      ],
+      "alertLevel": "HIGH"
```
[the remark this answers is not in the document]
Comment (mikiodehartj1) [reply rule-packs/mitre-attck-execution-attack-paths.json:348]: Corrected

File: rule-packs/mitre-attck-execution-attack-paths.json [hunk 9]
```diff
@@ -94,5 +94,257 @@
...
+    },
+    {
+      "name": "execution-user-execution-malicious-image-audit-image",
+      "description": "M1047 - Audit images deployed within the environment to ensure they do not contain any malicious components.",
+      "queries": [
+        {
+          "name": "query0",
+          "query": "FIND Container THAT !HAS Assessment",
```
[the remark this answers is not in the document]
Comment (mikiodehartj1) [reply rule-packs/mitre-attck-execution-attack-paths.json:344]: Deleting question - Can't find a working example

File: rule-packs/mitre-attck-execution-attack-paths.json (hunk 4 above)
Comment (mikiodehartj1) [reply rule-packs/mitre-attck-execution-attack-paths.json]: adding

**Review by SeaBlooms — APPROVED** [review]

**Pull request merged**